composer.json

```
@@ -4,39 +4,50 @@
     "description": "Wishlist plugin for Sylius.",
     "license": "MIT",
     "require": {
-        "php": "^8.1",
-        "sylius/sylius": "^1.13 || ^1.14",
-        "symfony/webpack-encore-bundle": "^1.15",
-        "dompdf/dompdf": "^2.0"
+        "php": "^8.2",
+        "sylius/sylius": "^2.0",
+        "dompdf/dompdf": "^2.0",
+        "sylius/twig-hooks": "^0.6",
+        "sylius/twig-extra": "^0.6",
+        "symfony/ux-autocomplete": "^2.17",
+        "symfony/ux-live-component": "^2.20",
+        "symfony/ux-twig-component": "^2.20",
+        "symfony/stimulus-bundle": "^2.12",
+        "symfony/webpack-encore-bundle": "^2.1"
     },
     "require-dev": {
-        "behat/behat": "^3.7",
-        "behat/mink-selenium2-driver": "~1.6.0",
+        "behat/behat": "^3.14",
+        "behat/mink-selenium2-driver": "^1.7",
         "bitbag/coding-standard": "^3.0.0",
-        "dmore/behat-chrome-extension": "^1.3",
-        "dmore/chrome-mink-driver": "^2.7",
-        "gedmo/doctrine-extensions": "^3.9",
+        "dmore/behat-chrome-extension": "^1.4",
         "friends-of-behat/mink": "^1.8",
         "friends-of-behat/mink-browserkit-driver": "^1.4",
-        "friends-of-behat/mink-debug-extension": "^2.0",
+        "friends-of-behat/mink-debug-extension": "^2.0.0",
         "friends-of-behat/mink-extension": "^2.4",
         "friends-of-behat/page-object-extension": "^0.3",
         "friends-of-behat/suite-settings-extension": "^1.0",
         "friends-of-behat/symfony-extension": "^2.1",
         "friends-of-behat/variadic-extension": "^1.3",
+        "gedmo/doctrine-extensions": "^3.9",
+        "league/flysystem-bundle": "^3.3",
+        "nelmio/alice": "^3.10",
+        "nyholm/psr7": "^1.8",
         "phpspec/phpspec": "^7.0",
         "phpunit/phpunit": "^9.5",
-        "symfony/browser-kit": "^5.4 || ^6.0",
-        "symfony/debug-bundle": "^5.4 || ^6.0",
-        "symfony/dotenv": "^5.4 || ^6.0",
-        "symfony/intl": "^5.4 || ^6.0",
-        "symfony/web-profiler-bundle": "^5.4 || ^6.0",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
-        "symfony/dependency-injection": "^5.4 || ^6.0",
-        "league/flysystem-bundle": "2.4.0",
         "sylius/mailer-bundle":  "^1.8 || ^2.0@beta",
```

This one could be bumped up if it is needed
```suggestion
        "sylius/mailer-bundle":  "^2.1",
```